eras/conway/test-suite/src/Test/Cardano/Ledger/Conway/Examples/Consensus.hs

```diff
@@ -202,5 +201,5 @@ exampleConwayNewEpochState =
     emptyPParams
     (emptyPParams & ppCoinsPerUTxOByteL .~ CoinPerByte (Coin 1))
 
-exampleConwayGenesis :: Crypto c => ConwayGenesis c
-exampleConwayGenesis = def
+exampleConwayGenesis :: ConwayGenesis StandardCrypto
+exampleConwayGenesis = expectedConwayGenesis
```

Comment:
Note in the changelog that the value has changed, because that will affect consensus golden tests. Also requires a major version bump, since this is semantic change.